pages/index.tsx

```diff
@@ -61,7 +70,14 @@ export default function Home() {
       </Head>
       <main className={styles.main}>
         <NavBar />
-      <Map />
+        <div className='info-container'>
+          <div className='poi-container' >
+            <button className='test-btn' onClick={fetchPlaces}>Test button</button>
+            <button className='test-btn' onClick={infoFlip}>Test flip button</button>
```

Comment:
[for reference] For the future we need to create a button component that we can pass a function for it to call. This will make it re-useable

Reply:
Made ticket will be working on that next